packages/mobile/ios/Podfile.lock

```diff
@@ -15,9 +15,9 @@ PODS:
     - React-Core (= 0.66.5)
     - React-jsi (= 0.66.5)
     - ReactCommon/turbomodule/core (= 0.66.5)
-  - ffmpeg-kit-ios-https (5.1.LTS)
-  - ffmpeg-kit-react-native/https-lts (5.1.0):
-    - ffmpeg-kit-ios-https (= 5.1.LTS)
+  - ffmpeg-kit-ios-https (4.5.1.LTS)
```

Comment:
So I think we don't want this downgrade? To fix remove your pods, checkout pod lock from main, and then run pod install again

Reply:
Might need to also do the thing for package lock first as well. Lmk if you pairing can help on this

Reply:
You said you looked into this and mine was good. Is this correct or not?